doc/modules/preprocessing.rst

```diff
@@ -445,28 +445,96 @@ values, either using the mean, the median or the most frequent value of
 the row or column in which the missing values are located. This class
 also allows for different missing values encodings.
 
+There is an option ``add_missing_indicator`` which appends the indicator
```

Comment:
I think it could be phrased in a passive voice...

The `add_missing_indicator` option can be used to append an indicator matrix to the imputed data `X`. The individual columns of the indicator matrix map the the features which had missing values and is used to infer whether that particular feature of a sample has been imputed or not.

@jnothman